addons/pos_hr/static/src/js/chrome.js

```diff
@@ -95,7 +95,7 @@ odoo.define('pos_hr.chrome', function (require) {
             'args': {
                 label: _lt('Lock'),
                 action: function() {
-                    this.gui.show_screen('login');
+                    this.chrome.return_to_login_screen();
```

Comment:
why changin this?

Reply:
The function will be override when using employee in blackbox. Checking if the user clocked out before showing the login screen.